salt/metalk8s/volumes/prepared/files/metalk8s-sparse-volume@.service

```diff
@@ -0,0 +1,13 @@
+[Unit]
+Description=Setup MetalK8s sparse loop device %I
+RequiresMountsFor=/var/lib/metalk8s/storage/sparse
+
+[Service]
+Type=oneshot
+ExecStart=/sbin/losetup --find --partscan "/var/lib/metalk8s/storage/sparse/%i"
+ExecStop=/usr/local/bin/metalk8s-volume-clean-up "%i"
```

Comment:
I really think this should go in `/usr/local/bin/libexec`, not `/usr/local/bin`. It's not some executable supposed to be regularly executed by a system user.

Also, `cleanup` :wink:

Reply:
> It's not some executable supposed to be regularly executed by a system user.

`/usr/local/sbin` then no ?

Reply:
Nope, `sbin` is for executables to be used by a superuser. This script is only ever to be executed by `systemd`, not by any regular/interactive user, hence `libexec`.

Reply:
Done.